index.html

```diff
@@ -0,0 +1,26 @@
+<html>
+<head>
+    <title>HPO</title>
```

Comment:
This needs to be `Hyper Parameter Optimization as a Service`

Reply:
Done